src/qt/walletview.h

```diff
@@ -35,26 +35,26 @@ class WalletView : public QStackedWidget
     Q_OBJECT
 
 public:
-    explicit WalletView(const PlatformStyle *platformStyle, QWidget *parent);
+    explicit WalletView(WalletModel* wallet_model, const PlatformStyle* platformStyle, QWidget* parent);
     ~WalletView();
 
     /** Set the client model.
         The client model represents the part of the core that communicates with the P2P network, and is wallet-agnostic.
     */
     void setClientModel(ClientModel *clientModel);
-    WalletModel *getWalletModel() { return walletModel; }
-    /** Set the wallet model.
-        The wallet model represents a bitcoin wallet, and offers access to the list of transactions, address book and sending
-        functionality.
-    */
-    void setWalletModel(WalletModel *walletModel);
+    WalletModel* getWalletModel() const { return walletModel; }
```

Comment:
Maybe worth adding `noexcept` and `[[nodiscard]]` too? Or at least just `noexcept`, as it's not "heavyweight" operation to ask for "nodsicardness", but still...

Reply:
Also forgot to add, `WalletModel *walletModel;` pointer itself now can be `const`.

Reply:
> Also forgot to add, `WalletModel *walletModel;` pointer itself now can be `const`.

```
In file included from qt/moc_walletview.cpp:9:
./qt/walletview.h:45:50: error: cannot initialize return object of type 'WalletModel *' with an lvalue of type 'const WalletModel *const'
    WalletModel* getWalletModel() const { return walletModel; }
                                                 ^~~~~~~~~~~
1 error generated
```

Reply:
Not `const WalletModel *` but `WalletModel *const walletModel`. Pointer itself `const`, not the object pointed to.